MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -436,6 +440,26 @@ private void ExitModule(SessionBase session)
             }
         }
 
+        /// <summary>
+        ///     Invokes routine registered as LONROU during MAJORBBS->REGISTER_MODULE() call on only
+        ///     the selected Rlogin module, specified by ModuleIdentifier.
+        ///
+        ///     Executes on the given channel once after a user successfully logs in.
+        /// </summary>
+        private void ProcessLONROU_FromRlogin(SessionBase session)
+        {
+            bool.TryParse(_configuration["Module.DoLoginRoutine"], out var doLoginRoutine);
+
+            if (doLoginRoutine)
```

Comment:
`OutputEnabled = doLoginRoutine`, this config setting should only impact the output of the login routine not if it should be run. We should run login routines no matter what (as some modules require it), but give the user the option to suppress the spam.

We need to set the `OutputEnabled = doLoginRoutine` logic in the current `ProcessLONROU` as well, to always run but suppress spam if config is set to true.

Also -- is there a risk of running the login routine multiple times?

Reply:
I think we want this always false in the RLogin case at least. I've moved the doLoginRoutine check into `ProcessLONROU` though

There should be no risk of running this routine twice. It's protected by the SessionState, so unless somehow the state gets reset back to LoginRoutines, it won't get called again.